src/parser/content-evaluator-module.ts

```
@@ -542,14 +542,32 @@ export class ContentEvaluatorModule extends BaseModule {
+      try {
+        res = await callLlm(payload, this.context);
+      } catch (error) {
+        // Fallback for providers that only support json_object.
+        res = await callLlm(
+          {
+            ...payload,
+            response_format: { type: "json_object" },
+          },
+          this.context
+        );
+      }
```

ESLint

[error] Handle this exception or don't catch it at all.

(sonarjs/no-ignored-exceptions)